cmd/kube-proxy/app/server.go

```diff
@@ -514,13 +517,63 @@ type ProxyServer struct {
 	Client        clientset.Interface
 	Broadcaster   events.EventBroadcaster
 	Recorder      events.EventRecorder
-	Conntracker   Conntracker // if nil, ignored
 	NodeRef       *v1.ObjectReference
 	HealthzServer healthcheck.ProxierHealthUpdater
+	Hostname      string
```

Comment:
this could be internal variables

Reply:
you mean `hostname` rather than `Hostname`? yeah, the whole struct is wrong that way

Reply:
yep yep